docs/dg/dev/backend-development/data-manipulation/data-publishing/implement-publish-and-synchronization.md

```diff
@@ -239,6 +224,12 @@ $productAbstractEntity->setColorCode("#FFFFFF");
 $productAbstractEntity->save();
 ```
 
+Or events can be triggered manually by using the `EventFacade`:
```

Comment:
Can we provide an example of one or more cases when the events should be triggered manually?
And let's also mention about `publish:trigger-events ` command